{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"requests.exceptions.SSLError: HTTPSConnectionPool(host='sdcore-nms.10.0.0.5.nip.io', port=443): Max retries exceeded with url: /network-configuration (Caused by SSLError(SSLError(1, '[SSL: TLSV1_UNRECOGNIZED_NAME] tlsv1 unrecognized name (_ssl.c:1007)')))","children":[{"name":"test_given_external_hostname_configured_for_traefik_when_calling_sdcore_nms_then_configuration_tabs_are_available","uid":"672c3cd63b24c018","parentUid":"7d21e9de5def277ce377129a39521abf","status":"broken","time":{"start":1749782725641,"stop":1749782735565,"duration":9924},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":["asyncio","abort_on_fail"]}],"uid":"7d21e9de5def277ce377129a39521abf"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}